deployment/src/test/java/com/datastax/oss/quarkus/runtime/api/session/QuarkusCqlSessionTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.api.session;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.datastax.dse.driver.api.core.cql.reactive.ReactiveRow;
+import com.datastax.oss.driver.api.core.cql.ExecutionInfo;
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
+import com.datastax.oss.driver.internal.core.cql.DefaultExecutionInfo;
+import com.datastax.oss.quarkus.CassandraTestBase;
+import com.datastax.oss.quarkus.runtime.api.reactive.MutinyReactiveResultSet;
+import io.quarkus.test.QuarkusUnitTest;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.reactivex.subscribers.TestSubscriber;
```

Comment:
It's a bit odd to use rxJava here. Mutiny has its own test utilities:

```
    <dependency>
      <groupId>io.smallrye.reactive</groupId>
      <artifactId>mutiny-test-utils</artifactId>
      <version>0.5.0</version>
      <scope>test</scope>
    </dependency>
```

Then we could rewrite the assertions as:

 ```
private void validateReactiveRowNotEmpty(Multi<ReactiveRow> reactiveRowMulti) {
  List<ReactiveRow> rows =
      reactiveRowMulti
          .subscribe()
          .withSubscriber(MultiAssertSubscriber.create(Long.MAX_VALUE))
          .await()
          .assertCompletedSuccessfully()
          .items();
  assertThat(rows).isNotEmpty();
}
```

But I see one small problem: we need to declare the version of mutiny-test-utils explicitly, and it could get eventually out of sync with the Mutiny version used in Quarkus. Is this a problem? I'm hesitating.

Reply:
Why do you think that this is better than simply collecting all records?
```
  MultiSubscribe<ReactiveRow> result = reactiveRowMulti.subscribe();
    List<ReactiveRow> collect = result.asIterable().stream().collect(Collectors.toList());
    assertThat(collect).isNotEmpty();
```
It does not require any subscriber.
Regarding `validateExecutionInfoNotEmpty`, we can rework it to mutiny testing util.